chain/src/types.rs

```diff
@@ -57,30 +57,32 @@ pub struct Tip {
 	pub height: u64,
 	/// Last block pushed to the fork
 	pub last_block_h: Hash,
-	/// Block previous to last
+	/// Previous block
 	pub prev_block_h: Hash,
 	/// Total difficulty accumulated on that fork
 	pub total_difficulty: Difficulty,
 }
 
 impl Tip {
-	/// Creates a new tip at height zero and the provided genesis hash.
-	pub fn new(gbh: Hash) -> Tip {
+	/// TODO - why do we have Tip when we could just use a block header?
```

Comment:
Block header comes with all sorts of things (mmr roots, PoW, etc) that we don't care about when tracking just a chain tip?